recipe/meta.yaml

```diff
@@ -23,13 +23,11 @@ package:
 source:
   # Using github as source for mlflow-skinny which does not exist on PyPI
   url: https://github.com/mlflow/mlflow/archive/refs/tags/v{{ version }}.zip
-  sha256: d101bf639b7671408d67d357f4af207e8655a41d50ca3befa951026d49751e39
+  sha256: 910ad1e2fce93b412063637d465e9291c99678a3bbccbf2b4f7736887321f44b
 
 build:
   number: 0
-  entry_points:
-    - mlflow=mlflow.cli:cli
-  skip: true  # [py<38 or s390x]
+  skip: true  # [py<39 or s390x]
```

Comment:
The skip is still `py<38` isn't it?  https://github.com/mlflow/mlflow/blob/master/pyproject.toml#L23